core/state_transition.go

```diff
@@ -397,13 +396,14 @@ func (st *StateTransition) distributeTxFees() error {
 
 	// Pay gateway fee to the specified recipient.
 	if st.msg.GatewayFeeRecipient() != nil {
-		err := st.creditFee(*st.msg.GatewayFeeRecipient(), st.msg.GatewayFee(), st.msg.FeeCurrency())
-		if err != nil {
+		log.Debug("Crediting gateway fee", "recipient", *st.msg.GatewayFeeRecipient(), "amount", st.msg.GatewayFee(), "feeCurrency", st.msg.FeeCurrency())
```

Comment:
Please remove these log messages, or at least move them to `Trace`. From experience, these really clutter up the logs.